V3 docs - Position notifications with example for central aligning notifications to center top (#10084)
laravel-filament/filament: packages/notifications/docs/05-customizing-notifications.md

Notifications have dedicated CSS classes you can hook into to apply your own styling. Open the inspector in your browser to find out which classes you need to target.

## Positioning notifications

> You can configure the alignment of the notifications with a service provider, for example adding the below to the boot method on: AppServiceProvider.php would align the notifications top middle of the browser.

Reviewer: ```suggestion
You can configure the alignment of the notifications in a service provider or middleware, by calling `Notifications::alignment()` and `Notifications::verticalAlignment()`.
```

Please also add a list of relevant options for these enums, like we do in other parts of the docs

```php

>         Notifications::alignment(Alignment::Center);

Reviewer: Please fix indentation and add imports for the class and enums

Author: Sorry Dan, been a bit swamped. Thanks for the cleanup! I was using github online for these.
